File: src/libs/ThirdParty.VBSuppliers/Properties/ExpediaRapid/SerializableClasses/Book/BookRequestRoom.vb
Imports Newtonsoft.Json
Namespace ExpediaRapid.SerializableClasses.Book

    Public Class BookRequestRoom

        <JsonProperty("title")>
        Public Property Title As String

        <JsonProperty("given_name")>
        Public Property GivenName As String

        <JsonProperty("family_name")>
        Public Property FamilyName As String

        <JsonProperty("email")>
        Public Property Email As String

        <JsonProperty("phone")>
        Public Property Phone As Phone

        <JsonProperty("smoking")>
        Public Property Smoking As Boolean

        <JsonProperty("special_request")>
        Public Property SpecialRequest As String

    End Class

End Namespace

File: src/libs/iVectorOne.Suppliers/ExpediaRapid/SerializableClasses/Book/BookRequestRoom.cs
namespace iVectorOne.Suppliers.ExpediaRapid.SerializableClasses.Book
{
    using Newtonsoft.Json;


    public class BookRequestRoom
    {

        [JsonProperty("given_name")]
        public string GivenName { get; set; }

        [JsonProperty("family_name")]
        public string FamilyName { get; set; }

        [JsonProperty("smoking")]
        public bool Smoking { get; set; }

        [JsonProperty("special_request")]
        public string SpecialRequest { get; set; }

    }

}